pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-overview-of-modules.adoc

```diff
@@ -0,0 +1,12 @@
+[id="overview-of-modules_{context}"]
+
+= Overview of modules
+[role="_abstract"]
+This section describes what a module is in the context of {ProductShortName} and how {ProductShortName} tells modules apart.
```

Comment:
I'd recommend avoiding wording such as 'this section' because this is a standalone user story.